src/css/parse.js

```diff
@@ -1537,23 +1527,21 @@ function checkKeyframesSelector(i) {
   return i - start;
 }
 
-
 /**
  * Get a single keyframe selector
  * @returns {Node}
  */
 function getKeyframesSelector() {
-  let keyframesSelectorType = NodeType.KeyframesSelectorType;
-  let selectorType = NodeType.SelectorType;
-
-  let token = tokens[pos];
-  let line = token.ln;
-  let column = token.col;
+  const keyframesSelectorType = NodeType.KeyframesSelectorType;
+  const selectorType = NodeType.SelectorType;
+  const token = tokens[pos];
+  const line = token.ln;
+  const column = token.col;
   let content = [];
 
   if (token.keyframesSelectorType === 1) {
     content.push(getIdent());
-  } else {
+  } else if (token.keyframesSelectorType === 2) {
```

Comment:
Why do we need to check for `2` if we know that at this point the type is always `2`?